### PR TITLE
Removed unused imports

### PR DESCRIPTION
**Pull request by MitchellGale**
### Description
Removed unused imports in spotless.

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by GumpacG — APPROVED** [review]
LGTM but DCO fails and there's merge conflicts :(

File: common/src/test/java/org/opensearch/sql/common/authinterceptors/AwsSigningInterceptorTest.java [hunk 1]
```diff
@@ -7,8 +7,6 @@
 
 package org.opensearch.sql.common.authinterceptors;
 
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
```
Comment (Yury-Fridlyand) [inline common/src/test/java/org/opensearch/sql/common/authinterceptors/AwsSigningInterceptorTest.java:10]: Probably, remove one blank line too to have only one blank between package declaration and imports.

File: integ-test/src/test/java/org/opensearch/sql/ppl/ConvertTZFunctionIT.java [hunk 2]
```diff
@@ -7,15 +7,13 @@
...
-
+  
   import java.io.IOException;
 
   import static org.opensearch.sql.legacy.TestsConstants.TEST_INDEX_DATE;
   import static org.opensearch.sql.util.MatcherUtils.rows;
   import static org.opensearch.sql.util.MatcherUtils.schema;
-  import static org.opensearch.sql.util.MatcherUtils.verifyDataRows;
-  import static org.opensearch.sql.util.MatcherUtils.verifySchema;
+    import static org.opensearch.sql.util.MatcherUtils.verifySchema;
```
Comment (Yury-Fridlyand) [inline integ-test/src/test/java/org/opensearch/sql/ppl/ConvertTZFunctionIT.java:16]:
```suggestion
import static org.opensearch.sql.util.MatcherUtils.verifySchema;
```

File: integ-test/src/test/java/org/opensearch/sql/ppl/ConvertTZFunctionIT.java [hunk 3]
```diff
@@ -7,15 +7,13 @@
 
   import org.json.JSONObject;
   import org.junit.Test;
-  import org.opensearch.sql.legacy.SQLIntegTestCase;
-
+
```
Comment (Yury-Fridlyand) [inline integ-test/src/test/java/org/opensearch/sql/ppl/ConvertTZFunctionIT.java:10]:
```suggestion
```

File: legacy/src/main/java/org/opensearch/sql/legacy/esdomain/LocalClusterState.java [hunk 4]
```diff
@@ -6,7 +6,6 @@
 
 package org.opensearch.sql.legacy.esdomain;
 
-import static org.opensearch.common.settings.Settings.EMPTY;
```
Comment (Yury-Fridlyand) [inline legacy/src/main/java/org/opensearch/sql/legacy/esdomain/LocalClusterState.java:9]:
```suggestion
```

File: legacy/src/main/java/org/opensearch/sql/legacy/rewriter/matchtoterm/TermFieldScope.java [hunk 5]
```diff
@@ -7,7 +7,6 @@
 package org.opensearch.sql.legacy.rewriter.matchtoterm;
 
 import java.util.*;
```
Comment (Yury-Fridlyand) [inline legacy/src/main/java/org/opensearch/sql/legacy/rewriter/matchtoterm/TermFieldScope.java:9]: Not related by this fix, but do you have `spotless` rule for star imports?

File: prometheus/src/test/java/org/opensearch/sql/prometheus/storage/PrometheusStorageFactoryTest.java [hunk 6]
```diff
@@ -282,4 +281,3 @@ void createDataSourceSuccessWithHostnameRestrictions() {
   }
 
 }
-
```
Comment (Yury-Fridlyand) [inline prometheus/src/test/java/org/opensearch/sql/prometheus/storage/PrometheusStorageFactoryTest.java:285]: typo?

File: legacy/src/main/java/org/opensearch/sql/legacy/rewriter/matchtoterm/TermFieldScope.java (hunk 5 above)
Comment (MitchellGale) [reply legacy/src/main/java/org/opensearch/sql/legacy/rewriter/matchtoterm/TermFieldScope.java:9]: What would you want to happen, would you want star imports to be converted to the correct imports?

File: integ-test/src/test/java/org/opensearch/sql/ppl/ConvertTZFunctionIT.java (hunk 3 above)
Comment (MitchellGale) [reply integ-test/src/test/java/org/opensearch/sql/ppl/ConvertTZFunctionIT.java]: Addressed in 65df31b37e9b2766d0a5fd4c31745fdc3638d4af

File: integ-test/src/test/java/org/opensearch/sql/ppl/ConvertTZFunctionIT.java (hunk 2 above)
Comment (MitchellGale) [reply integ-test/src/test/java/org/opensearch/sql/ppl/ConvertTZFunctionIT.java]: Addressed in 65df31b37e9b2766d0a5fd4c31745fdc3638d4af

File: legacy/src/main/java/org/opensearch/sql/legacy/esdomain/LocalClusterState.java (hunk 4 above)
Comment (MitchellGale) [reply legacy/src/main/java/org/opensearch/sql/legacy/esdomain/LocalClusterState.java]: Addressed in 65df31b37e9b2766d0a5fd4c31745fdc3638d4af

File: legacy/src/main/java/org/opensearch/sql/legacy/rewriter/matchtoterm/TermFieldScope.java (hunk 5 above)
Comment (Yury-Fridlyand) [reply legacy/src/main/java/org/opensearch/sql/legacy/rewriter/matchtoterm/TermFieldScope.java:9]: Yes, I think our checkstyle config rejects star imports

**Review by Yury-Fridlyand — APPROVED** [review]

**Pull request closed without merging**